fixtures/vuejs3-typescript/App.vue

```diff
@@ -1,12 +1,13 @@
 <template>
   <div id="app">
-    <img src="./assets/logo.png">
+    <img :src="logoPath">
     <hello></hello>
   </div>
 </template>
 
 <script lang="ts">
 import Hello from './components/Hello.vue'
+import logoPath from './assets/logo.png';
```

Comment:
This and the `package.json` files are unrelated, but fix the tests. I'm not sure what's going on - it's super complex. It relates to this commit - https://github.com/vuejs/vue-loader/commit/21725a4ebc9c8d7f8a590d700017759327e21c2e - and vue-loader 16.7.0. Our testing setup is very vanilla, but for some reason, the `<img src="./assets/logo.png">` no longer worked. I have no idea how to fix that - and it's so strange, that, for now, I'm going to assume it's some edge-case issue specific to our app or some bug that will be fixed upstream.